android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarLocationPuck.kt

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.maps.testapp.auto.car
+
+import android.annotation.SuppressLint
+import androidx.car.app.CarContext
+import com.mapbox.maps.R
+import com.mapbox.maps.extension.style.expressions.dsl.generated.literal
+import com.mapbox.maps.plugin.LocationPuck2D
+import com.mapbox.maps.plugin.LocationPuck3D
+import com.mapbox.maps.toJson
+
+/**
+ * Provides car location puck definitions.
+ */
+internal object CarLocationPuck {
+  /**
+   * 3D location puck with the real world size.
+   */
+  val duckLocationPuckHighZoom = LocationPuck3D(
+    modelUri = "https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF-Embedded/Duck.gltf",
```

Comment:
Duck goes prod 💪

Reply:
:heart_eyes:

Reply:
Used a link to the public repo that hosting the glTF samples instead of embedding this file to our repo.